src/index.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import combinedReducers from './reducers/index'
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-
+let store = createStore(combinedReducers,composeWithDevTools)
```

Comment:
Thanks for setting this up! It takes care of our Store set up issue. I'll link it to this PR